db/re/item_combo_db.txt

```diff
@@ -2742,3 +2741,16 @@
 24742:24584:24585:24586:24587:24588,{ bonus2 bIgnoreDefRaceRate,RC_All,70; bonus2 bIgnoreMDefRaceRate,RC_All,70; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-70; bonus bLongAtkRate,15; bonus bDelayrate,-15; bonus bVariableCastrate,-10; bonus2 bSkillCooldown,"RL_HEAT_BARREL",-5000; bonus2 bSkillAtk,"RL_FIREDANCE",15; bonus2 bSkillAtk,"RL_R_TRIP",15; bonus2 bSkillAtk,"RL_FIRE_RAIN",15; }
 24743:24584:24585:24586:24587:24588,{ bonus2 bSkillCooldown,"SP_SOULCURSE",-1500; bonus2 bMagicAtkEle,Ele_All,15; bonus2 bIgnoreDefRaceRate,RC_All,70; bonus2 bIgnoreMdefRaceRate,RC_All,70; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-70; bonus bVariableCastrate,-10; bonus2 bSkillCooldown,"SP_SOULREAPER",-5000; bonus2 bSkillAtk,"SP_CURSEEXPLOSION",15; bonus2 bSkillAtk,"SP_SPA",15; bonus2 bSkillAtk,"SP_SWHOO",15; }
 24744:24584:24585:24586:24587:24588,{ bonus2 bMagicAtkEle,Ele_All,15; bonus2 bIgnoreDefRaceRate,RC_All,70; bonus2 bIgnoreMdefRaceRate,RC_All,70; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-70; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-70; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-70; bonus bShortAtkRate,15; bonus bDelayrate,-15; bonus bVariableCastrate,-10; bonus2 bSkillAtk,"RK_IGNITIONBREAK",15; bonus2 bSkillAtk,"SO_PSYCHIC_WAVE",15; }
+2114:15001:5122,{ bonus bStr,2; bonus bDef,5; bonus bMdef,5; if(BaseClass==Job_Swordman) bonus bDef,6; }
+2114:15001:5124,{ bonus bDef,2-getequiprefinerycnt(EQI_HAND_L)-getequiprefinerycnt(EQI_HEAD_TOP); bonus bMdef,5+getequiprefinerycnt(EQI_HAND_L)+getequiprefinerycnt(EQI_HEAD_TOP); }
+2455:2559:5544:15007,{ bonus bBaseAtk,20; bonus bMatk,20; bonus bAspdRate,10; }
```

Comment:
I can't find the description giving theses bonus. Link?

Reply:
Hidden bonus.  Reference from thRO

![image](https://user-images.githubusercontent.com/60462533/145932515-21545be2-96b8-4296-bc39-0835a0133fb4.png)

https://youtu.be/05PDJOlXT90

https://youtu.be/PpcWQhTXi4g